docs/supportedIntegrations.md

```diff
@@ -6,28 +6,29 @@ The table below provides a comprehensive list of the integrations and PingFedera
+| [HTML Form Login](https://docs.pingidentity.com/csh?Product=pf-latest&topicname=xvy1564003022890.html)                                                     | :heavy_check_mark:    | :heavy_check_mark:  |
+| [ID DataWeb Integration Kit](https://docs.pingidentity.com/bundle/integrations/page/ndg1577481773402.html)                                                 | :heavy_check_mark:    | :heavy_check_mark:  |
+| [Identifier First](https://docs.pingidentity.com/csh?Product=pf-latest&topicname=iek1564003022460.html)                                                    | :heavy_check_mark:    | :heavy_check_mark:  |
+| [Jamf Integration Kit](https://docs.pingidentity.com/bundle/integrations/page/wyg1612987868528.html)                                                       | :heavy_check_mark:    | :heavy_check_mark:  |
+| [LinkedIn Cloud Identity Connector](https://docs.pingidentity.com/bundle/integrations/page/pwm1563995028175.html)                                          | :heavy_check_mark:    | :heavy_check_mark:  |
+| [Microsoft Cloud Identity Connector](https://docs.pingidentity.com/bundle/integrations/page/set1623964507521.html)                                         | :heavy_check_mark:    | :heavy_check_mark:  |
+| [One-Time Passcode Integration Kit](https://docs.pingidentity.com/bundle/integrations/page/hdv1589402808052.html)                                          | :heavy_check_mark:    |                     |
+| [PingOne MFA Integration Kit](https://docs.pingidentity.com/bundle/integrations/page/cal1599060087050.html)                                                | :heavy_check_mark:    | :heavy_check_mark:  |
+| [PingOne Risk Management Integration Kit](https://docs.pingidentity.com/bundle/integrations/page/fnd1592583282135.html)                                    | :heavy_check_mark:    | :heavy_check_mark:  |
+| [PingOne Verify Integration Kit](https://docs.pingidentity.com/bundle/integrations/page/elk1607471062724.html)                                             | :heavy_check_mark:    | :heavy_check_mark:  |
+| [RSA SecurID Integration Kit](https://docs.pingidentity.com/bundle/integrations/page/ikv1563995044731.html)                                                | :heavy_check_mark:    | :heavy_check_mark:  |
+| [ThreatMetrix Integration Kit](https://docs.pingidentity.com/bundle/integrations/page/nnx1593461471877.html)                                               | :heavy_check_mark:    | :heavy_check_mark:  |
+| [Twitter Cloud Identity Connector](https://docs.pingidentity.com/bundle/integrations/page/cav1563995059579.html)                                           | :heavy_check_mark:    | :heavy_check_mark:  |
+| [Symantec VIP Integration Kit](https://docs.pingidentity.com/bundle/integrations/page/fvd1563995058360.html)                                               | :heavy_check_mark:    | :heavy_check_mark:  |
```

Comment:
I just realized that the table is alphabetical! 😮  Would you mind putting Symantec VIP Integration Kit and Azure AD Identity Protection Integration Kit in order as well? (also remove the weird azure-ad-identity-protection in the end, not sure how it got there) Thank you!